csharp/src/Microsoft.ML.OnnxRuntime/PinnedOnnxValue.cs

```diff
@@ -0,0 +1,72 @@
+using Microsoft.ML.OnnxRuntime.Tensors;
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.ML.OnnxRuntime
+{
+    /// <summary>
+    /// Represents an Onnx Value with its underlying buffer pinned
+    /// </summary>
+    public class PinnedOnnxValue : IDisposable
```

Comment:
The name 'PinnedOnnxValue' doesn't make it obvious when to use this API --  both NamedOnnxValue and DisposableNamedOnnxValue do pinning as well, so the term Pinned doesn't differentiate it much.

Since the main use case is for buffer reuse,  should this be called something like  ReusableOnnxValue or FixedBufferOnnxValue or something more descriptive? Ideally the name should let users know immediately how/when to use this API.

Reply:
renamed to FixedBufferOnnxValue